boot/zephyr/main.c

```diff
@@ -498,7 +502,9 @@ void main(void)
     BOOT_LOG_INF("Bootloader chainload address offset: 0x%x",
                  rsp.br_image_off);
 
+#if !defined(MCUBOOT_DIRECT_XIP)
     BOOT_LOG_INF("Jumping to the first image slot");
```

Comment:
or maybe BOOT_LOG_INF("Jumping to the image slot") for direct-xip mode?

Reply:
Will change the message as suggested.